README.md

```diff
@@ -52,6 +51,74 @@ To reload the extension after some changes, open [`chrome://extensions`](chrome:
 
 To use the local API instead of the prod one, change `ORBIT_ROOT_URL` in `orbit-helper.js` to your ngrok tunnel address.
 
+## Contributing
+
+Pull requests & issues welcome!
+
+### Accessibility
+
+Accessibility is a fundamental part of web design, and something we consider critical to the further development of this extension. Future contirbutions should aim to meet [WCAG2.1 AA](https://www.w3.org/TR/WCAG21/) - if you notice any areas where we're failing, please raise them as issues or submit PRs to address them.
+
+Common issues to check:
+
+1. Verify all functionality is usable with keyboard-only control
+2. Open your screenreader (VoiceOver is built-in for macOS, or Narrator in windows) & confirm navigating across the widget provides clear, contextual, and useful information akin to what is shown visually.
```

Comment:
Couple of minor suggestions:

- Can you add links to some documentation about VoiceOver and Narrator?
- Windows should be capitalized
- Use `and` rather than `&`